pybossa/forms/forms.py

```diff
@@ -800,7 +800,7 @@ class UserPrefMetadataForm(Form):
     profile = TextAreaField(
         lazy_gettext("<div>Task Preferences</div>"
                      "<div style='color:red; font-weight:normal; font-size:12px;'>"
-                     "Must not include sensitive or personally identifiable information.</div>"),
+                     "Must not include sensitive nor personally identifiable information, e.g., name, email address, phone number, UUID, race, gender, health or financial information.</div>"),
```

Comment:
I think if you want to use "nor" here it has to be "Must include neither sensitive information nor . . .". Otherwise it should probably be "Must not include sensitive or personally identifiable. . . ." [For example, when you use the word not, the structure “not A or B” is correct. ](https://www.mentalfloss.com/article/639689/when-use-nor)

https://www.mentalfloss.com/article/639689/when-use-nor

Reply:
Changed to "or" via https://github.com/bloomberg/pybossa/pull/597/commits/052030696749a0bd176c7f06083057ad3582d242